api-flask/Dockerfile

```diff
@@ -7,3 +7,4 @@ COPY requirements.txt .
 RUN pip3 install --upgrade pip
 RUN pip3 install -r requirements.txt
 RUN pip3 install greenlet==0.4.17
+RUN pip3 install flask-restx
```

Comment:
This should be added to `requirements.txt` instead

Reply:
At first, I did try to add this in the requirement.txt but for some reason, it didn't work. I can try one more time if you would like :)

Reply:
The you will need to rebuild the docker. Let me know if you can't get it to work and I can have a look